<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signals and Slots</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="titlelabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Test a different signals and slots notation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="stringNotationButton">
       <property name="text">
        <string>String Notation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="functorNotationButton">
       <property name="text">
        <string>Functor Notation with regular slot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="functorLambdaNotationButton">
       <property name="text">
        <string>Functor Notation with lambda function</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="stringNotationLabel">
       <property name="text">
        <string>Changed from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="functorNotationLabel">
       <property name="text">
        <string>Changed from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="functorLambdaNotationLabel">
       <property name="text">
        <string>Changed from:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>stringNotationButton</tabstop>
  <tabstop>functorNotationButton</tabstop>
  <tabstop>functorLambdaNotationButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
